packages/security/vue.config.js

```diff
@@ -0,0 +1,41 @@
+const BannerPlugin = require('webpack').BannerPlugin
```

Comment:
Misschien een idee om een algemene vue.config.js te hebben, waar iedere package uit overerft? Er zit redelijk wat herhaling in.

Reply:
nee er zijn allemaal verschillende voor oude projecten